blocks/cards/cards.css

```diff
@@ -2,30 +2,156 @@
   list-style: none;
   margin: 0;
   padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
-  grid-gap: 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 50px;
 }
 
-.cards > ul > li {
-  border: 1px solid var(--highlight-background-color);
-  background-color: var(--background-color)
+.cards.border > ul {
+  justify-content: flex-start;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.card {
+  width: calc(25% - 15px); /* default 4 columns */
+  backface-visibility: hidden;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
+
+.card.with-border {
```

Comment:
I wrote the CSS for the card without prepending it with `.cards` in the idea that we might reuse it for the carousels from https://www.walgreens.com/store/catalog/shopLanding, as they look pretty similar.